src/index.js

```diff
@@ -1,7 +1,30 @@
 import './style.css';
+import { addScore } from './modules/addScore.js';
+
+import { getScores } from './modules/getScores.js';
+
+import { display } from './modules/displayScore.js';
 
 const submitForm = document.getElementById('submit');
+const refresh = document.querySelector('button');
+const score = document.getElementById('score');
 
-submitForm.addEventListener('click', (e) => {
+getScores().then((response) => display(response.result));
+
+submitForm.addEventListener('click', async (e) => {
   e.preventDefault();
+  const name = e.target.parentElement.firstElementChild;
+  const score = name.nextElementSibling;
+  await addScore(name.value, Number(score.value));
```

Comment:
[ Optional Change ] This change won't stop your app from being approved but will improve the experience of your app.

- [ ] It would be nice if you could learn the input fields after adding a score. Add these codes after line 18 to clear the input fields. 👍 

```javascript
  name.value = ''
  score.value = ''
```

Reply:
@AmaduKamara thanks for the review and the suggestion. I implemented in my last commit.